sprocs/assessment_instances_select_log.sql

```diff
@@ -24,6 +24,30 @@ AS $$
 BEGIN
     RETURN query
         WITH
+        ai_group_users AS (
```

Comment:
Add a comment that this is not just current group users (which would be cleaner) but all users who were ever in the group?

Reply:
Re "cleaner", I just saw a real-world example in Mariana's course where a student created a group with themselves in it, did the assessment, and then left the group. If we didn't show all past members too, we'd end up with an empty assessment log in that case, which seems undesirable from an instructor's perspective.

Reply:
Done in 4d37fde. I also inlined my above comment in case someone ever tries to be clever and change that code 😂

Reply:
Haha, nice!